src/main/java/org/fundacionjala/sfdc/pageobjects/contacts/SFCNewModifyPage.java

```diff
@@ -125,31 +111,48 @@ public void setContactPhoneTextField(final String contactPhoneTextField) {
 
     /**
      * Method to set contact las Name attribute.
+     *
      * @param contactHomePhoneTextField string value.
      */
     public void setContactHomePhoneTextField(final String contactHomePhoneTextField) {
         this.contactHomePhoneTextField.clear();
         this.contactHomePhoneTextField.sendKeys(contactHomePhoneTextField);
     }
+
+
     /**
-     * Method to do click "save" button.
+     * @param values contact field
+     * @return map of contact fields.
      */
-    public void clickSaveContactButton() {
-        getSaveContactButton().click();
+    public Map<ContactInputs, Value> getStrategyStepMap(final Map<ContactInputs, String> values) {
+        Map<ContactInputs, Value> strategyMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)